RecoTracker/MkFitCMS/standalone/tkNtuple/WriteMemoryFile.cc

```diff
@@ -512,6 +513,7 @@ int main(int argc, char* argv[]) {
     t->SetBranchAddress("ph2_yz", &ph2_yz);
     t->SetBranchAddress("ph2_zz", &ph2_zz);
     t->SetBranchAddress("ph2_zx", &ph2_zx);
+    t->SetBranchAddress("ph2_clustSize", &ph2_clustSize);
```

Comment:
this is a recent addition, perhaps protect 

```suggestion
    if (t->GetBranch("ph2_clustSize"))
      t->SetBranchAddress("ph2_clustSize", &ph2_clustSize);
```